languages/sr_RS/Zend_Validate.php

```diff
@@ -0,0 +1,308 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license    http://framework.zend.com/license/new-bsd     New BSD License
+ */
+
+/**
+ * sr_RS-Revision: 28.October.2015
+ */
+return array(
+    // Zend\Authentication\Validator\Authentication
+    "Invalid identity" => "Погрешан идентитет",
+    "Identity is ambiguous" => "Идентитет је сумњив",
+    "Invalid password" => "Погрешна лозинка",
+    "Authentication failed" => "Неуспела аутентификација",
+
+    // Zend\I18n\Validator\Alnum
+    "Invalid type given. String, integer or float expected" => "Погрешан тип унет. Очекиван је текст, цео број или стварни број",
+    "The input contains characters which are non alphabetic and no digits" => "Унос садржи карактере који нису алфабетски и нису бројеви",
+    "The input is an empty string" => "Унет је празан текст",
+
+    // Zend\I18n\Validator\Alpha
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+    "The input contains non alphabetic characters" => "Унос садржи не-алфабетске карактере",
+    "The input is an empty string" => "Унет је празан текст",
+
+    // Zend\I18n\Validator\DateTime
+    "Invalid type given. String expected" => "Погрешан тип унет. Очекиван је текст",
+    "The input does not appear to be a valid datetime" => "Унет је неисправан датум и време",
```

Comment:
Hmm I think that `datetime` here is particular data type (expects well-known format). Your translation is _"date and time"_. Again, I'm all in favor of keeping original to avoid misunderstanding: `datetime`.